contrib/go-redis/redis.v7/redis_test.go

```diff
@@ -457,6 +447,8 @@ func TestWithContext(t *testing.T) {
 		}
 	}
 
+	assert.Equal(ctx1, client1.Context())
+	assert.Equal(ctx2, client2.Context())
```

Comment:
Same here. We don't really have to test this, since these methods are not implemented by us. You're testing that the child/parent relationship is ok and that's all that's needed.